conda/base/context.py

```diff
@@ -77,14 +77,19 @@ def default_python_default():
 
 
 def default_python_validation(value):
-    if value and len(value) == 3 and value[1] == '.':
-        try:
-            value = float(value)
-            if 2.0 <= value < 4.0:
-                return True
-        except ValueError:  # pragma: no cover
-            pass
-    return "default_python value '%s' not of the form '[23].[0-9]'" % value
+    if value:
+        if len(value) == 3 and value[1] == '.':
+            try:
+                value = float(value)
+                if 2.0 <= value < 4.0:
+                    return True
+            except ValueError:  # pragma: no cover
+                pass
+    else:
```

Comment:
@kalefranz I could be more explicit here like

`elif value is None or value == '':`